src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -6,8 +6,9 @@
 					<summary-card />
 					<!-- Borrower details -->
 				</div>
-				<div class="tw-col-span-12 md:tw-col-start-6 md:tw-col-span-6 lg:tw-col-span-5">
+				<div class="tw-col-span-12 md:tw-col-start-2 md:tw-col-span-10 lg:tw-col-span-5">
 					<!-- Powered by lenders info -->
+					<lend-cta />
 				</div>
```

Comment:
Instead of passing props into the lendCta, I just moved the query inside the lendCta component to fetch the data we need for that component and it doesn't need to be prefetched.